### DISCUSSION TITLE
Undiscord delay

### DISCUSSION THREAD

**Discussion by shandarovska** [victornpb/undiscord#600] [1. Questions & Answers] [answered]
I'm trying to delete 65k+ messages and I'm keeping the delay (3000ms,search delay)-(2850,delete delay) and its going quite slowly, it has deleted around 20k for the past 48h but it kept stopping randomly at times. Any solutions?

**Comment by viiper2000** [comment, marked as answer]
I'd recommend raising the search delay to at least 12000-20000 ms, and the delete delay down about 1800 ms. You'll still get occasional hiccups with finding pages and it will be slower, but it _should_ be much more consistent than having a very short delay between finding pages.

Additionally, try tinkering with the before and after dates in the settings so you aren't accidentally trying to delete messages that have already been deleted.
